ChangeLog.rst

```diff
@@ -1,3 +1,11 @@
+libfuse 3.16.x (tbd)
+====================
```

Comment:
Should be 3.17

Reply:
Yes, I'd already spotted that and fixed in the next push.